examples/find.js

```diff
@@ -1,20 +1,24 @@
-const dht = require('../')
+const DHT = require('../')
 const crypto = require('crypto')
 
 const hex = process.argv[2]
-const node = dht({ ephemeral: true, bootstrap: ['localhost:10001'] })
+const node = new DHT({ ephemeral: true, bootstrap: ['localhost:10001'] })
 
-node.query('values', Buffer.from(hex, 'hex'))
-  .on('data', function (data) {
+run()
+
+async function run () {
+  const q = node.query(Buffer.from(hex, 'hex'), 'values')
+
+  for await (const data of q) {
     if (data.value && sha256(data.value).toString('hex') === hex) {
       // We found the value! Destroy the query stream as there is no need to continue.
```

Comment:
The comment mentions about "destroy" but no destroy happens here.

Reply:
The break destroys it :)